### PR TITLE
Adding extra premium plugin changes to 5.1 RNs

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: _includes/release-notes/51new-features.md [hunk 1]
```diff
@@ -195,6 +195,42 @@ Fixes for inline dialogs and menus have been included to:
...
+### Enhanced Media Embed
+
+The {{site.productname}} 5.1 release includes **Enhanced Media Embed** 2.2.4.
+
+**Enhanced Media Embed** 2.2.4 has been updated to include a touch event listener to allow embedded media to execute on mobile platforms.
+
+### Format Painter
+The {{site.productname}} 5.1 release includes **Format Painter** 1.2.0.
+
+**Format Painter** 1.2.0 has been updated to support the mobile platforms. The **Format Painter** will now function as expected for mobile users.
```
Comment (blackcathikari) [inline _includes/release-notes/51new-features.md]: "the mobile platforms" - should it include "the"?

Comment (tylerkelly13) [reply _includes/release-notes/51new-features.md]: no... no it should not :+1:

Comment (tylerkelly13) [reply _includes/release-notes/51new-features.md]: fixed

File: _includes/release-notes/51new-features.md [hunk 2]
```diff
@@ -195,6 +195,42 @@ Fixes for inline dialogs and menus have been included to:
 
 ## Premium Plugin changes
 
+### Advcode
```
Comment (lnewson) [inline _includes/release-notes/51new-features.md]: This should probably be `Advanced Code` or `Advanced Code Editor` I guess?

Comment (tylerkelly13) [reply _includes/release-notes/51new-features.md]: fixed

**Pull request merged**